### PR TITLE
Add MobileVitV2 to doctests

### LINKED ISSUE #24763: sample code in for mobilevitv2 is not working
### System Info

- `transformers` version: 4.31.0.dev0
- Platform: macOS-10.16-x86_64-i386-64bit
- Python version: 3.10.11
- Huggingface_hub version: 0.14.1
- Safetensors version: 0.3.1
- PyTorch version (GPU?): 2.0.0 (False)
- Tensorflow version (GPU?): not installed (NA)
- Flax version (CPU?/GPU?/TPU?): not installed (NA)
- Jax version: not installed
- JaxLib version: not installed
- Using GPU in script?: no
- Using distributed or parallel set-up in script?:no

### Who can help?

_No response_

### Information

- [X] The official example scripts
- [ ] My own modified scripts

### Tasks

- [X] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

run sample code from [MobileViTV2ForImageClassification](https://huggingface.co/docs/transformers/v4.30.0/en/model_doc/mobilevitv2#transformers.MobileViTV2ForImageClassification)

throws an error

ValueError: Unrecognized image processor in apple/mobilevitv2-1.0-imagenet1k-256. Should have a `image_processor_type` key in its preprocessor_config.json of config.json, or one of the following `model_type` keys in its config.json: align, beit, bit, blip, blip-2, bridgetower, chinese_clip, clip, clipseg, conditional_detr, convnext, convnextv2, cvt, data2vec-vision, deformable_detr, deit, deta, detr, dinat, donut-swin, dpt, efficientformer, efficientnet, flava, focalnet, git, glpn, groupvit, imagegpt, instructblip, layoutlmv2, layoutlmv3, levit, mask2former, maskformer, mgp-str, mobilenet_v1, mobilenet_v2, mobilevit, mobilevitv2, nat, oneformer, owlvit, perceiver, pix2struct, poolformer, regnet, resnet, sam, segformer, swiftformer, swin, swin2sr, swinv2, table-transformer, timesformer, tvlt, upernet, van, videomae, vilt, vit, vit_hybrid, vit_mae, vit_msn, xclip, yolos

### Expected behavior

sample code should not throw an error

### PR DESCRIPTION
**Pull request by amyeroberts**
# What does this PR do?

Adds MobileVitV2 to the doctests. The example snippet wasn't working because the model's config files pointed to an image processor that doesn't exist. This adds the models to the doctests so that this is caught. 

Also removes a duplicate line in image_processing_auto.py

Fixes #24763 (partially)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: src/transformers/models/auto/image_processing_auto.py [hunk 1]
```diff
@@ -75,7 +75,6 @@
         ("mgp-str", "ViTImageProcessor"),
         ("mobilenet_v1", "MobileNetV1ImageProcessor"),
         ("mobilenet_v2", "MobileNetV2ImageProcessor"),
-        ("mobilenet_v2", "MobileNetV2ImageProcessor"),
```
Comment (ydshieh) [inline src/transformers/models/auto/image_processing_auto.py:78]: nice!

**Review by ydshieh — APPROVED** [review]
LGTM. Thanks.

This PR will fail due to the PR documentation test job if I understand correctly the issue. (So further change should be done before merge)

(We should probably start to progressive running the doctest and add them to the file)

**Comment by ydshieh** [comment]
> ValueError: Files in `utils/documentation_tests.txt` are not in alphabetical order.

@amyeroberts  

You are the one creating this check 😆

**Comment by amyeroberts** [comment]
image

**Comment by ydshieh** [comment]
Oh, I am wrong! The PR doctest is not triggered as this PR doesn't change modeling file. Great!

**Pull request merged**